### ISSUE TITLE
Problem with POST method

### ISSUE THREAD

**Issue by Kikobeats** [balderdashy/sails.io.js#20]
I don't why but when I use `POST` method for create a new model the response object have the verb `updated` instead of `created` using the function `on`

Also the backend server say:

```
warn: `Model.subscribe()` called by a non-socket request. Only requests originating from a connected socket may be subscribed. Ignoring...
```

![Imgur](http://i.imgur.com/K50l8xq.png)

**Comment by intoxicated** [comment]
I got this warning when I attempted to make one to many association between existing model via PUT request. Can I just ignore it ???

**Comment by crobinson42** [comment]
Post your code 

Cory

**Comment by jvanalst** [comment]
I'm also getting this warning after introducing a One-To-Many Relationship.

```javascript
/**
* Equipment-Category.js
*
* @description :: TODO: You might write a short summary of how this model works and what it represents here.
* @docs        :: http://sailsjs.org/#!documentation/models
*/

module.exports = {

  attributes: {
    name: {
      type: 'string',
      unique: true,
      required: true
    },
    equipment: {
      collection: 'equipment',
      via: 'category'
    }
  }
};
```

```javascript
/**
* Equipment.js
*
* @description :: TODO: You might write a short summary of how this model works and what it represents here.
* @docs        :: http://sailsjs.org/#!documentation/models
*/

module.exports = {
  attributes: {
    tagNumber: {
      type: 'int',
      required: false
    },
    name: {
      type: 'string',
      size: 127,
      required: true
    },
    description: {
      type: 'text',
      required: true
    },
    allowBookings: {
      type: 'string',
      enum: ['yes', 'no'],
      required: true
    },
    category: {
      model: 'equipment-category',
      required: true
    }
  }
};
```

```
curl -X POST "http://localhost:1337/api/equipment/create?name=First%20Category&category=1&allowBookings=yes&description=Moo"
```

**Comment by mattblang** [comment]
We are seeing this when we do a `PUT /model/id` blueprint route, after adding a `one-many relationship` to the `model`.

**Comment by hamzaniazi91** [comment]
anyone with the solution?

**Comment by sailsbot** [comment]
Thanks for posting, @Kikobeats.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by Ggleny** [comment]
and the solution is ........?

**Comment by sgress454** [comment]
This thread was kinda hard to answer--multiple different issues being posted on top of each other.  Nobody here could recreate the original issue, or the one with one-to-many relationships (perhaps it's not an issue in v0.11+?).  For those who posted things like:

```
socket.post('/user/create?firstName=Cory&lastName=Jameson', function(user){ console.dir(user)});
```

I will say that those [shortcut routes](http://sailsjs.org/documentation/reference/blueprint-api#?blueprint-routes) should _always_ be GET requests.  They are also intended for development only and should be turned off in production.
